Source/carthage/Formatting.swift

```diff
@@ -0,0 +1,103 @@
+//
+//  Formatting.swift
+//  Carthage
+//
+//  Created by J.D. Healy on 1/29/15.
+//  Copyright (c) 2015 Carthage. All rights reserved.
+//
+
+import Commandant
+import Foundation
+import LlamaKit
+import PrettyColors
+import ReactiveCocoa
+
+/// Wraps or passes through a string.
```

Comment:
Can you please clarify this documentation? What does it mean to “wrap” a string? (This is somewhat rhetorical, because I know the answer, but I'd love for the documentation to express it.)